src/commands/setup.js

```diff
@@ -200,9 +200,13 @@ function createWidget(logger, api, settings, widgetDefaults) {
     const widget = Object.assign(
         {
             app_id: settings.app.id,
-            description: settings.app.description,
             source: 'test',
-            title: settings.app.name,
+            translations: {
+                en: {
```

Comment:
Does this also support japanese translations. If so we should set those here aswell.

Also it would be great if we had a path in config.json that would tell which language the name or description are.
If possible use fallback in order not to break existing widgets.

You could do something like this:

```js{4}
const translations = {
    en: {
        title: settings.app.en && settings.app.en.name || settings.app.name,
        description: settings.app.en && settings.app.en.description || settings.app.description
    },
    ja: {
        title: settings.app.ja && settings.app.ja.name || settings.app.name,
        description: settings.app.ja && settings.app.ja.description || settings.app.description
    }
}
```

This would allow us to introduce japanese translations to our widgets titles and descriptions.

Reply:
I think I've implemented something similar to what you've asked for.  There are now two new fields in the questions and two new for the Japanese data.   I've defaulted to English should the Ja fields be left blank.